plugins/events/create.js

```diff
@@ -219,9 +219,16 @@ module.exports = () => ({
                                         )
                                         // User has good permissions, create an event
                                         .then(() =>
-                                            scm.getCommitSha(scmConfig).then(sha => {
-                                                payload.sha = sha;
-                                            })
+                                            scm
+                                                .getCommitSha(scmConfig)
+                                                .then(sha => {
+                                                    payload.sha = sha;
+                                                })
+                                                .catch(err => {
+                                                    if (err.status === 404) {
+                                                        throw boom.notFound(err);
```

Comment:
If you use the following code, you may be able to handle errors other than 404 with boom.
`throw boom.boomify(err, {statusCode: err.status})`

Reply:
Thx. I fixed it.